### PR TITLE
Add reusable metric definitions to changelog

### PR DESCRIPTION
**Pull request by nihalathanikkal**

### CODE REVIEW

File: fern/docs/content/change-log/2024-04.mdx [hunk 1]
```diff
@@ -2,6 +2,17 @@
 title: Changelog | April, 2024
 ---
 
+## Reusable Metric Definitions
+
+_April 30th, 2024_
+
+Introducing reusable Evaluation Metrics!
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-04.mdx:9]:
```suggestion
Introducing Reusable Evaluation Metrics!
```

File: fern/docs/content/change-log/2024-04.mdx [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+
+Evaluation Metrics can now be shared across your Test Suites making it easier for you to consistently test and evaluate your Prompt / Workflow performance.
+Define a suite of custom metrics tailored to your business logic and use case to save time and ensure standardized evaluations.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-04.mdx:12]:
```suggestion
Define a suite of custom metrics tailored to your business logic and use-case to save time and ensure standardized evaluation criteria.
```

File: fern/docs/content/change-log/2024-04.mdx [hunk 3]
```diff
@@ -2,6 +2,17 @@
 title: Changelog | April, 2024
 ---
 
+## Reusable Metric Definitions
+
+_April 30th, 2024_
+
+Introducing reusable Evaluation Metrics!
+
+Evaluation Metrics can now be shared across your Test Suites making it easier for you to consistently test and evaluate your Prompt / Workflow performance.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-04.mdx:11]:
```suggestion
Evaluation Metrics can now be shared across your Test Suites making it easier for you to consistently test and evaluate your Prompt / Workflow quality.
```

File: fern/docs/content/change-log/2024-04.mdx [hunk 4]
```diff
@@ -2,6 +2,17 @@
 title: Changelog | April, 2024
 ---
 
+## Reusable Metric Definitions
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-04.mdx:5]:
```suggestion
## Reusable Evaluation Metrics
```

**Review by noanflaherty — APPROVED** [review]
Woo!

**Review by ashleeradka — APPROVED** [review]

**Pull request merged**